### PR TITLE
SQL Perfmon counters - synced queries from v2 to all db types

### PR DESCRIPTION
**Pull request by avinash-nigam**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

Includes changes :
- replicated perfmon counters list from v2 query to queries across all database types

### CODE REVIEW

**Review by denzilribeiro — CHANGES_REQUESTED** [review]
Is there someone that needs  the counter below? There are hardly any customers using buffer pool extensions. If not needed please remove that and will aprrove.
Extension free pages | Total number of free cache pages in the buffer pool extension file.

**Comment by avinash-nigam** [comment]
> Is there someone that needs the counter below? There are hardly any customers using buffer pool extensions. If not needed please remove that and will aprrove.
> Extension free pages | Total number of free cache pages in the buffer pool extension file.

@denzilribeiro I've removed the buffer manager counters

**Review by denzilribeiro — APPROVED** [review]
Looks good

**Comment by avinash-nigam** [comment]
@ssoroka and @reimda - could you please help merge this PR as well? Thank you.

File: plugins/inputs/sqlserver/sqlqueriesV2.go
Comment (ssoroka) [inline plugins/inputs/sqlserver/sqlqueriesV2.go:480]: why do we remove these two?

Comment (avinash-nigam) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: These were added recently as part of PR - https://github.com/influxdata/telegraf/pull/8120, however these counters are rarely used (Denzil's comment above mention the same), hence were removed.

Comment (avinash-nigam) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: @ssoroka - any thoughts on the above? if there are no concerns, could you please help merge the same? thank you!

**Comment by avinash-nigam** [comment]
@ssoroka and @reimda - Could you please help merge this PR if there are no concerns? Thank you.

**Comment by avinash-nigam** [comment]
> @ssoroka and @reimda - Could you please help merge this PR if there are no concerns? Thank you.

@ssoroka and @reimda - Any thoughts on the above?

**Comment by masree** [comment]
> > @ssoroka and @reimda - Could you please help merge this PR if there are no concerns? Thank you.
> 
> @ssoroka and @reimda - Any thoughts on the above?

@ssoroka  and @reimda - Can you please help merge PR if no concerns? Thank you!

**Comment by denzilribeiro** [comment]
@danielnelson  / @ssoroka  / @sjwang90   can someone help merge this or are there pending issues that need to be addressed?

**Comment by sjwang90** [comment]
@denzilribeiro We'll get this reviewed and let you know if there's anything else before it gets merged. Thanks!

**Comment by masree** [comment]
@sjwang90 Can you help take a look of this PR and merge it if no concerns?

File: plugins/inputs/sqlserver/sqlqueriesV2.go
Comment (ssoroka) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: any chance that someone is depending on them?

Comment (denzilribeiro) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: Unlikely this was supposed to be added to here instead by mistake was recently added to V2.

Comment (masree) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: @ssoroka / @sjwang90 - Any thoughts on the above? Please help merge the PR if no concerns. Thanks!

Comment (srebhan) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: @ssoroka your decision here. #8120 was merged on Sept. 2020 and thus was release in 1.17.

My question to @denzilribeiro is if it was intended to be added here, why do you now remove it. In general, what is the cost of keeping those two in there?

**Review by srebhan — COMMENTED** [review]
I think the removal of the two properties needs to be agreed on by someone from InfluxData. Alternatively, you could leave those lines in if it is not prohibitively costly (my preferred solution).

File: plugins/inputs/sqlserver/sqlqueriesV2.go
Comment (denzilribeiro) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: @srebhan  - to be honest should have not been added as rarely ( < 0.5%) do SQL Server folks use it. Removal was more from perspective of unnecessary space, not opposed to leaving it but if 99.9% are not using it then why? But you guys' call

Comment (ssoroka) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: It's possible if someone is using it this could break a chart. I'm on the fence here, but if it's all the same to everyone let's just leave the fields in and they can always drop the ones they don't want.

Comment (avinash-nigam) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:480]: @srebhan @ssoroka - Let me revert the change of removing the two counters, rest of the PR would remain as is. I'll push the change and ping back here.

Comment (avinash-nigam) [reply plugins/inputs/sqlserver/sqlqueriesV2.go]: @srebhan @ssoroka - The two counters' removal has been reverted. If there are no other concerns, it'd be very helpful to get this merged. Thank you.

**Comment by avinash-nigam** [comment]
> I think the removal of the two properties needs to be agreed on by someone from InfluxData. Alternatively, you could leave those lines in if it is not prohibitively costly (my preferred solution).

@srebhan - Reverted the removal of the two counters.

**Review by srebhan — APPROVED** [review]
Looks good to me.

**Comment by avinash-nigam** [comment]
@sjwang90 @ssoroka, could you please help merge this?

**Review by helenosheaa — APPROVED** [review]

**Pull request merged**
